core/variant/variant_call.cpp

```diff
@@ -2522,12 +2522,40 @@ static void _register_variant_builtin_methods() {
 	_VariantCall::add_variant_constant(Variant::VECTOR3, "ZERO", Vector3(0, 0, 0));
 	_VariantCall::add_variant_constant(Variant::VECTOR3, "ONE", Vector3(1, 1, 1));
 	_VariantCall::add_variant_constant(Variant::VECTOR3, "INF", Vector3(INFINITY, INFINITY, INFINITY));
+
+#ifndef DISABLE_DEPRECATED
 	_VariantCall::add_variant_constant(Variant::VECTOR3, "LEFT", Vector3(-1, 0, 0));
 	_VariantCall::add_variant_constant(Variant::VECTOR3, "RIGHT", Vector3(1, 0, 0));
 	_VariantCall::add_variant_constant(Variant::VECTOR3, "UP", Vector3(0, 1, 0));
 	_VariantCall::add_variant_constant(Variant::VECTOR3, "DOWN", Vector3(0, -1, 0));
 	_VariantCall::add_variant_constant(Variant::VECTOR3, "FORWARD", Vector3(0, 0, -1));
 	_VariantCall::add_variant_constant(Variant::VECTOR3, "BACK", Vector3(0, 0, 1));
+#endif
+	_VariantCall::add_variant_constant(Variant::VECTOR3, "CAMERA_LEFT", Vector3(-1, 0, 0));
+	_VariantCall::add_variant_constant(Variant::VECTOR3, "CAMERA_RIGHT", Vector3(1, 0, 0));
+	_VariantCall::add_variant_constant(Variant::VECTOR3, "CAMERA_UP", Vector3(0, 1, 0));
+	_VariantCall::add_variant_constant(Variant::VECTOR3, "CAMERA_DOWN", Vector3(0, -1, 0));
+	_VariantCall::add_variant_constant(Variant::VECTOR3, "CAMERA_FORWARD", Vector3(0, 0, -1));
+	_VariantCall::add_variant_constant(Variant::VECTOR3, "CAMERA_BACKWARD", Vector3(0, 0, 1));
+
+	_VariantCall::add_variant_constant(Variant::VECTOR3, "MODEL_LEFT", Vector3(1, 0, 0));
+	_VariantCall::add_variant_constant(Variant::VECTOR3, "MODEL_RIGHT", Vector3(-1, 0, 0));
+	_VariantCall::add_variant_constant(Variant::VECTOR3, "MODEL_TOP", Vector3(0, 1, 0));
+	_VariantCall::add_variant_constant(Variant::VECTOR3, "MODEL_BOTTOM", Vector3(0, -1, 0));
+	_VariantCall::add_variant_constant(Variant::VECTOR3, "MODEL_FRONT", Vector3(0, 0, 1));
+	_VariantCall::add_variant_constant(Variant::VECTOR3, "MODEL_BACK", Vector3(0, 0, -1));
+
+	_VariantCall::add_variant_constant(Variant::VECTOR3, "VIEW_LEFT", Vector3(-1, 0, 0));
+	_VariantCall::add_variant_constant(Variant::VECTOR3, "VIEW_RIGHT", Vector3(1, 0, 0));
+	_VariantCall::add_variant_constant(Variant::VECTOR3, "VIEW_TOP", Vector3(0, 1, 0));
+	_VariantCall::add_variant_constant(Variant::VECTOR3, "VIEW_BOTTOM", Vector3(0, -1, 0));
+	_VariantCall::add_variant_constant(Variant::VECTOR3, "VIEW_FRONT", Vector3(0, 0, 1));
+	_VariantCall::add_variant_constant(Variant::VECTOR3, "VIEW_BACK", Vector3(0, 0, -1));
+
+	_VariantCall::add_variant_constant(Variant::VECTOR3, "NORTH", Vector3(0, 0, -1));
+	_VariantCall::add_variant_constant(Variant::VECTOR3, "SOUTH", Vector3(0, 0, 1));
+	_VariantCall::add_variant_constant(Variant::VECTOR3, "EAST", Vector3(1, 0, 0));
+	_VariantCall::add_variant_constant(Variant::VECTOR3, "WEST", Vector3(-1, 0, 0));
```

Comment:
Would a ``GLOBAL_`` prefix make more sense here? ``Like GLOBAL_FORWARD``, ``GLOBAL_LEFT`` etc.

I wouldn't think to use NORTH, SOUTH etc. if I was looking for constants in global space.

Reply:
alright